packages/editor/src/client/i18n.ts

```diff
@@ -13,18 +13,18 @@ export interface DateFormatLocale {
   lng: string
   locale: Locale
 }
-const DateFormatLocales: DateFormatLocale[] = [
-  {lng: 'de', locale: deLocale},
-  {lng: 'en', locale: enLocale},
-  {lng: 'fr', locale: frLocale}
-]
+const DateFormatLocales = new Map<string, Locale>([
```

Comment:
If you use the `new Map` you don't need the `DateFormatLocale` typescript interface on line 12 anymore.

Reply:
And in general normal javascript variables start with a lowercase letter.